pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>2.3</version>
+  </parent>
+  <artifactId>github-organization-folder</artifactId>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
Probably want this to be `1.0-beta-1-SNAPSHOT`, assuming you will bump up the `workflow-multibranch` dependency to `1.15-beta-1`: it is illegal for a non-beta plugin release to depend on a beta release (though the UC is too dumb to warn you).